.github/workflows/build.yml

```diff
@@ -213,3 +217,72 @@ jobs:
 
       - name: "PHPStan"
         run: "make phpstan"
+
+  mutation-testing:
+    name: "Mutation Testing"
+    runs-on: "ubuntu-latest"
+    needs: ["tests", "static-analysis"]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "8.2"
+          - "8.3"
+          - "8.4"
+        operating-system: [ubuntu-latest]
+
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v5
+
+      - name: "Install PHP"
```

Comment:
Please use the composite action, same way it's used here (the commit first moves "Checkout build-infection" right after "Checkout", then we can use a local path to the composite action): https://github.com/phpstan/phpstan-doctrine/pull/704